src/FINDOLOGIC/Api/Responses/Xml21/Properties/Filter/Item/CategoryItem.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace FINDOLOGIC\Api\Responses\Xml21\Properties\Filter\Item;
+
+use FINDOLOGIC\Api\Helpers\ResponseHelper;
+use FINDOLOGIC\Api\Responses\Xml21\Properties\Filter\CategoryFilter;
+use SimpleXMLElement;
+
+class CategoryItem extends Item
+{
+    /** @var Item[] $items */
```

Comment:
Sub-Items are always `CategoryItem`s. Please also update the docblock for the getter.